css/style.css

```diff
@@ -24,18 +33,25 @@ h1 {
   display: flex;
   flex-direction: column;
   align-items: center;
-  border: 2px solid blue;
-  margin: 15px;
-  margin-top: 25px;
+  border: 2px solid #53d562;
+  box-shadow: 0px 0px 7px 2px;
+  margin: 25px 15px 5px;
   width: 300px;
   height: 200px;
 }
 
 .question {
+  text-decoration: underline;
+  text-decoration-color: rgb(47, 200, 47);
+  text-underline-offset: 2px;
 }
 
-[type="button"] {
-  border: 2px solid black;
-  padding: 5px;
-  border-radius: 5px;
+.answer-button {
+  background-color: #000;
+  color: #08f223;
+  border: 2px solid #53d562;
+  padding: 2px 3px;
+  font-family: "space-mono", mono;
+  font-weight: 400;
+  font-style: normal;
```

Comment:
These values are inherited from the `body`, so there should be no need to declare them here (again).

Reply:
For some reason it didn't work so I had to put it into the button declaration again and then it worked